percona-server-mongodb-4.4/Dockerfile.k8s

```diff
@@ -1,4 +1,4 @@
-FROM redhat/ubi8-minimal
+FROM redhat/ubi8-minimal:8.9-1108
```

Comment:
shouldn't be the versioned changes also merged into latest? 
As for the best practices:

Best Practices:
For Development and Testing: Using latest can be beneficial for staying up-to-date with the newest features and fixes. However, be mindful of potential instability.
For Production: It's generally recommended to use specific version tags. This approach ensures stability and predictability. You should have a process in place to regularly update the version, ideally after testing the new version in a non-production environment.